spec/AutomatedLinkAcceptance+services.yaml

```diff
@@ -93,170 +105,191 @@ clients:
         individual:
       service:
         basic:
-          ## TODO: Add Clients specific to this application, but addressing all other applications
+
         individual:
-          ## TODO: Add Clients specific to this application
+
 
   - http-client:
       application-name: ApplicationLayerTopology
-      release-number: 1.0.0
-      uuid: xx-1-0-0-http-c-bm-alt-1-0-0-000
+      release-number: 2.0.0
+      uuid: ALA-1-0-0-http-c-alt-2-0-0-000
     tcp-client:
-      ip-address: 10.118.125.157
-      tcp-port: 1005
-      uuid: xx-1-0-0-tcp-c-bm-alt-1-0-0-000
+      remote-protocol: http
+      ip-address: 1.1.3.13
+      tcp-port: 3013
+      uuid: ALA-1-0-0-tcp-c-alt-2-0-0-000
     operation-clients:
       own-oam:
         basic:
           - service-name: /v1/update-all-ltps-and-fcs
-            uuid: xx-1-0-0-op-c-bm-alt-1-0-0-000
+            uuid: ALA-1-0-0-op-c-bm-alt-2-0-0-000
           - service-name: /v1/update-ltp
-            uuid: xx-1-0-0-op-c-bm-alt-1-0-0-001
+            uuid: ALA-1-0-0-op-c-bm-alt-2-0-0-001
           - service-name: /v1/delete-ltp-and-dependents
-            uuid: xx-1-0-0-op-c-bm-alt-1-0-0-002
+            uuid: ALA-1-0-0-op-c-bm-alt-2-0-0-002
           - service-name: /v1/update-fc
-            uuid: xx-1-0-0-op-c-bm-alt-1-0-0-003
+            uuid: ALA-1-0-0-op-c-bm-alt-2-0-0-003
           - service-name: /v1/update-fc-port
-            uuid: xx-1-0-0-op-c-bm-alt-1-0-0-004
+            uuid: ALA-1-0-0-op-c-bm-alt-2-0-0-004
           - service-name: /v1/delete-fc-port
-            uuid: xx-1-0-0-op-c-bm-alt-1-0-0-005
+            uuid: ALA-1-0-0-op-c-bm-alt-2-0-0-005
         individual:
       service:
         basic:
-          ## TODO: Add Clients specific to this application, but addressing all other applications
+
         individual:
-          ## TODO: Add Clients specific to this application
+
 
   - http-client:
       application-name: OperationKeyManagement
-      release-number: 1.0.0
-      uuid: xx-1-0-0-http-c-is-okm-1-0-0-000
+      release-number: 2.0.0
+      uuid: ALA-1-0-0-http-c-okm-2-0-0-000
     tcp-client:
-      ip-address: 10.118.125.157
-      tcp-port: 1006
-      uuid: xx-1-0-0-tcp-c-is-okm-1-0-0-000
+      remote-protocol: http
+      ip-address: 1.1.3.14
+      tcp-port: 3014
+      uuid: ALA-1-0-0-tcp-c-okm-2-0-0-000
     operation-clients:
       own-oam:
         basic:
         individual:
       service:
         basic:
-          ## TODO: Add Clients specific to this application, but addressing all other applications
+
         individual:
-          ## TODO: Add Clients specific to this application
+
 
   - http-client:
-      application-name: ## TODO: Add Name of application to be addressed
-      release-number: ## TODO: Add Release number of application to be addressed
-      uuid: ## TODO: Add UUID of the HttpClient
+      application-name: MicroWaveDeviceInventory
+      release-number: 1.0.0
+      uuid: mwdi-1-0-0-http-s-is-000
```

Comment:
- uuids for the http client and tcp client shall no longer have the service type (i.e. api segment: bm/im/bs/is) inside.  (The services inside the operation clients still need to have them.) 
- Also the uuids must not start with the app-ID of the mwdi, as this is your target application (which is missing in your uuid as well). 

change from ... to: 
| from | to |
|---|---|
|mwdi-1-0-0-http-s-is-000|ala-1-0-0-http-s-mwdi-1-0-0-000|
|mwdi-1-0-0-tcp-s-is-000|ala-1-0-0-tcp-s-mwdi-1-0-0-000|

Reply:
* Corrected the format of the uuids for http and tcp clients as indicated
* Also changed the ObjectType from server (s) to client (s), since I believe this was needed as well